spec/services/invoices/provider_taxes/void_service_spec.rb

```diff
@@ -109,6 +109,19 @@
       end
     end
 
+    context "when invoice is not voided" do
+      before { invoice.finalized! }
+
+      it "returns an error" do
+        result = void_service.call
+
+        aggregate_failures do
```

Comment:
same here, this is not required anymore.

Reply:
Updated